packages/services/src/staking-service/index.ts

```diff
@@ -3,10 +3,12 @@ import { ServiceImpl } from '@connectrpc/connect';
 
 import { validatorInfo } from './validator-info';
 import { validatorPenalty } from './validator-penalty';
+import { getValidatorInfo } from './get-validator-info';
 
 export type Impl = ServiceImpl<typeof StakingService>;
 
-export const stakingImpl: Pick<Impl, 'validatorInfo' | 'validatorPenalty'> = {
+export const stakingImpl: Pick<Impl, 'getValidatorInfo' | 'validatorInfo' | 'validatorPenalty'> = {
+  getValidatorInfo,
```

Comment:
Would kinda be nice if this was an `Omit<Impl, ...>`, so that it was kinda in our face what we have not implemented yet